circle.yml

```diff
@@ -6,6 +6,8 @@ stages:
     workDir: ~/go/src/github.com/palantir/godel
     machine:
       reusable: true
+      services:
+        - docker
```

Comment:
synced with @nmiyake. Decided on adding a new test tag for the distgo tests which require docker executable.